tests/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ foreach(test_name adc1 adc2 casscf-fzc-sp casscf-semi casscf-sa-sp ao-casscf-sp
                   fd-freq-energy fd-freq-energy-large fd-freq-gradient
                   fd-freq-gradient-large fd-gradient freq-isotope1 freq-isotope2 fnocc1 fnocc2
                   fnocc3 fnocc4 frac frac-ip-fitting frac-traverse ghosts gibbs matrix1
+                  ipi
```

Comment:
This is the wrong place -- instead needs to look like https://github.com/psi4/psi4/pull/1057/files#diff-3dea97423f83302e647a3bab62bf1157L267-L270 for the tests to be added. This can be the sole appearance for an `ENABLE_ipi` cmake var. What I'll do is move your two tests from ctest to tests/pytests/ . Pytest can easily check if ipi is available in the current env, whereas this ctest mechanism is looking at the env at configuration time.

Reply:
This took me longer than expected .. Now it should be at the correct place in the pytests location. Is this the correct location? The tests should work no matter if ipi is available or not so that I did not use the new variable `using_ipi` in the pytests.

The current tests mainly check the interaction with the rest of psi4 to see if the variables are still correctly passed on and all calculations are as expected. Those will be passed onto ipi, if it is installed as well and executed with correct serverdata.